util/ninja.py

```diff
@@ -1,7 +1,21 @@
 import html
+import numpy as np
 
 
 class Ninja:
+    @staticmethod
+    def float_range(lower, upper, step):
+        """
+        Float range function
+
+        :param int lower: lower point
+        :param int upper: upper point
+        :param float step: step value
+        :return: list of values by lower, upper and step values
+        :rtype: list
+        """
+        return list(np.arange(lower, upper, step))
```

Comment:
I believe, `list(range(X, Y, Z))` should return the same result.

Example,
```python
import numpy as np

assert list(np.arange(10, 100, 2)) == list(range(10, 100, 2))
```

Reply:
But `range` does not works with float step. I need `np.arange(5, 9.5, 0.5)` for instance [here](https://github.com/egemenzeytinci/foxandbox/pull/20/files#diff-9a86e485094db8a83366b208462a39788e2b3cbf598cad596c544dee74b569fdR43).